### PR TITLE
Update icons (last and final update)

### LINKED ISSUE #15401: Update toolbar overflow and sidebar icons to align with Firefox visual styles
We have to update some toolbar icons (primary UI) with https://github.com/mozilla/pdf.js/pull/15384#issuecomment-1236098185 in 106. This creates visual inconsistencies with overflow menu, find bar and side bar icons which now need a refresh.

This issue is for updating the icons from the overflow menu and sidebar in accordance with new Firefox icon styles:
- provide icon visual assets (UX)
- implement icon changes (engineering)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

**Comment by Snuffleupagus** [comment]
It seems that *some* of the SVGs now contain `fill-rule="evenodd" clip-rule="evenodd"` while others don't. Is there a reason for this inconsistency, and if not can we perhaps just remove that to have slightly smaller files?

(This might also apply to previously updated icons, so perhaps a separate commit to clean-up those as well?)

**Comment by calixteman** [comment]
> It seems that _some_ of the SVGs now contain `fill-rule="evenodd" clip-rule="evenodd"` while others don't. Is there a reason for this inconsistency, and if not can we perhaps just remove that to have slightly smaller files?
> 

I don't know.

> (This might also apply to previously updated icons, so perhaps a separate commit to clean-up those as well?)

Yep it'd be better to do the cleanup in a follow-up.

File: web/images/secondaryToolbarButton-rotateCcw.svg [hunk 1]
```diff
@@ -1 +1,10 @@
...
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_265_4500)">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3.4105 4.83612L4.77001 6.19601C5.06701 6.49201 4.85701 7.00001 4.43701 7.00001H0.862006C0.602006 7.00001 0.391006 6.78901 0.391006 6.52901V2.95401C0.391006 2.53401 0.899006 2.32401 1.19601 2.62101L2.32796 3.75328C3.67958 1.78973 5.9401 0.5 8.5 0.5C12.636 0.5 16 3.864 16 8C16 12.136 12.636 15.5 8.5 15.5C4.704 15.5 1.566 12.663 1.075 9H2.59C3.068 11.833 5.532 14 8.5 14C11.809 14 14.5 11.309 14.5 8C14.5 4.691 11.809 2 8.5 2C6.35262 2 4.46893 3.13503 3.4105 4.83612Z" fill="black"/>
+</g>
+<defs>
+<clipPath id="clip0_265_4500">
+<rect width="16" height="16" fill="white"/>
+</clipPath>
+</defs>
+</svg>
```
Comment (Snuffleupagus) [inline web/images/secondaryToolbarButton-rotateCcw.svg:10]: Compared to the `secondaryToolbarButton-rotateCw.svg` file, this one seem to contain a bunch of left-over stuff that I think we ought to remove.

**Comment by calixteman** [comment]
Finally I made the cleanup in this patch.

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you!

**Pull request merged**